pkgs/top-level/python-packages.nix

```
@@ -9277,6 +9277,8 @@ self: super: with self; {
 
   python-hl7 = callPackage ../development/python-modules/python-hl7 { };
 
+  python-hwdata = callPackage ../development/python-modules/python-hwdata { };
```

```suggestion
  python-hwdata = callPackage ../development/python-modules/python-hwdata {
    inherit (pkgs) hwdata;
  };
```

Let's make sure we always get the correct hwdata package, even if someone would package https://pypi.org/project/hwdata/.

---

Wow, the linked package does in fact link back to this one. So only now I noticed that the canonical name should be `hwdata`, not `python-hwdata`.

https://github.com/xsuchy/python-hwdata/blob/main/setup.py#L4

---

Should this be renamed `python3Packages.hwdata` then?

---

Yes, update pname, directory name and the name binding for callPackage.